### PR TITLE
nolo tlc ciel 13_4_1

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [sha:9e4b9e7c8d74d1d57196f4ebeee8b6d5c95cc0c8]{
 -- Page 10032, Offset 34269.
 -- ……頭の中で、最後の撃鉄が落ちた。
-...The final firing hammer drops in my head.
+...The final hammer drops within my head.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt:4]: I know for a fact that 撃鉄が落ちる has been used elsewhere to describe shiki gettin jiggy with it, so you might want to check how it was TL'd elsewhere

File: script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt [hunk 2]
```diff
@@ -171,17 +171,17 @@ With a sound of a footstep, a shadow leans over me.
...
 -- Page 365, Offset 34304. Mods: @n.
 -- 「良かった。これでお互いに罪はありません。
-"That's great. With this, neither of us are at sin.
+"That's good. With this, neither of us will have to bear any sin.
 }
 [sha:be4616049925acb52ce36b7ac9f2a64f5c5c0046]{
 -- Page 365, Offset 34305.
 -- 互いに殺し合うという条件なら、私たちの罪は相殺されます。あとは、生き残った者に罰が残るだけです」
-If we're both trying to kill each other, then our sins will be counteracted. After that, all that will remain is the retribution of the survivor."
+Now that we're both trying to kill each other, our sins will balance each other out. In the end, all that will remain is penance for the survivor."
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt:184]: now i'm no catholic, but i think you might want to email the pope rq about that logic and get his take

**Review by Avuxo — APPROVED** [review]
ok but he stabbed her "point", isn't that a little final for her to be able to fucking stab him and have a conversation?

File: script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt (hunk 2 above)
Comment (nolo16) [reply script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt:184]: i'm a scientologist so i cant email the pope

i CAN email tom cruise though

**Pull request merged**